sbt/src/sbt-test/source-dependencies/macro-annotation/macros/src/main/scala/Macros.scala

```diff
@@ -3,24 +3,24 @@ import scala.language.experimental.macros
 import scala.annotation.StaticAnnotation
 
 object HelloMacro {
-  def impl(c: Context)(annottees: c.Tree*): c.Tree = {
+  def impl(c: Context)(annotatees: c.Tree*): c.Tree = {
     import c.universe._
 
-    annottees match {
+    annotatees match {
       case (classDecl: ClassDef) :: Nil =>
-        val q"$mods class $name[..$tparams] $ctorMods(...$paramss) extends { ..$earlydefns } with ..$bases { $self => ..$body }" = classDecl
+        val q"$mods class $name[..$tparams] $ctorMods(...$params) extends { ..$earlydefns } with ..$bases { $self => ..$body }" = classDecl
         q"""
-        case class $name(...$paramss) extends ..$bases {
+        case class $name(...$params) extends ..$bases {
```

Comment:
When I see `params` I think of `List[X]`. When I see `paramss`, I think of `List[List[X]]`. That's fairly common convention for compiler code base.